src/block-management/block-holder.js

```diff
@@ -49,6 +50,14 @@ export default class BlockHolder extends React.Component<PropsType, StateType> {
 		return <View />;
 	}
 
+	_insertBlocksAfter( blocks: Array<Object> ) {
```

Comment:
In https://github.com/wordpress-mobile/gutenberg-mobile/pull/159/files#diff-2d50daf729e068ecebcce113d1ce7f9aR270, we pass the `insertBlocksAfter` method to all the rendered items so, I don't think we need to check whether the prop exists, right?

Then again, I think we don't even need this `_insertBlocksAfter` here as we can construct a wrapped method with just the `blocks` parameter right there in https://github.com/wordpress-mobile/gutenberg-mobile/pull/159/files#diff-2d50daf729e068ecebcce113d1ce7f9aR270.

For example we can have this in block-manager.js:
```
insertBlocksAfter={ ( blocks ) =>
	this.insertBlocksAfter.bind( this )( blocks, value.item.clientId )
}
```
WDYT?